custom/enikshay/integrations/nikshay/repeater_generator.py

```diff
@@ -0,0 +1,194 @@
+
+    # Could not locate this in any of the case types : episode, person, occurence
+    # tu_location = SQLLocation.objects.get(location_id=person_case_properties.get('tu_choice'))
+    # "tcode": tu_location.metadata['nikshay_code'],
+    # and pcategory
+    person_properites = {
+        "pname": person_case_properties.get('name', ''),
+        "pgender": gender_mapping.get(person_case_properties.get('sex', ''), ''),
+        "page": person_case_properties.get('age', ''),
+        "paddress": person_case_properties.get('current_address', ''),
+        "pmob": person_case_properties.get('contact_phone_number', ''),
+        "dotphi": person_case_properties.get('phi', ''),
+        "cname": person_case_properties.get('secondary_contact_name_address', ''),
+        "caddress": person_case_properties.get('secondary_contact_name_address', ''),
+        "cmob": person_case_properties.get('secondary_contact_phone_number', ''),
+    }
+
+    if state_location:
+        person_properites['scode'] = state_location.metadata.get('nikshay_code', '')
+    if district_location:
+        person_properites['dcode'] = district_location.metadata.get('nikshay_code')
+
+    # {'dcode': u'JLR', 'paddress': u'123, near asdf, , , Jalore, Rajasthan ', 'cmob': u'1234567890',
+    # 'pname': u'home visit', 'scode': u'RJ', 'dotphi': u'Test S1-C1-D1-T1 PHI 1', 'pmob': u'1234567890',
+    # 'cname': u'123', 'caddress': u'123', 'pgender': 'T', 'page': u'79'}
+    return person_properites
+
+
+def _get_episode_case_properties(episode_case_properties):
+    # Not Found
+    # "sitedetail": episode_site.get(episode_case_properties.get('site_choice', ''), ''),
+    if episode_site.get(episode_case_properties.get('site_choice', None)):
+        episode_site.get(episode_site.get(episode_case_properties.get('site_choice')))
+    episode_case_date = episode_case_properties.get('date_of_diagnosis', None)
+    if episode_case_date:
+        episode_date = datetime.datetime.strptime(episode_case_date, "%Y-%m-%d").date()
+    else:
+        episode_date = datetime.date.today()
+
+    episode_year = episode_date.year
+    episode_properties = {
+        "poccupation": occupation.get(
+            episode_case_properties.get('occupation', 'other'),
+            occupation['other']
+        ),
+        "pregdate": str(episode_date),
+        "ptbyr": str(episode_year),
+        "disease_classification": disease_classification.get(
+            episode_case_properties.get('disease_classification', ''),
+            ''
+        ),
+        "dotname": (episode_case_properties.get('treatment_supporter_first_name', '') +
+                    episode_case_properties.get('treatment_supporter_last_name', '')
+                    ),
+        "dotmob": episode_case_properties.get('treatment_supporter_mobile_number', ''),
+        # Can this mandatory field be made N/A if in case we don't collect this as in spec
```

Comment:
Can we just pass the designation that we have? Does it conflict with the expected values?